multisrc/src/main/java/eu/kanade/tachiyomi/multisrc/wpmangastream/WPMangaStreamGenerator.kt

```diff
@@ -33,6 +33,7 @@ class WPMangaStreamGenerator : ThemeSourceGenerator {
         SingleLang("Manga Pro Z", "https://mangaprotm.com", "ar", overrideVersionCode = 1),
         SingleLang("Manga Raw.org", "https://mangaraw.org", "ja", className = "MangaRawOrg", overrideVersionCode = 1),
         SingleLang("MangaIndonesia", "https://mangaindonesia.net", "id"),
+        SingleLang("Mareceh", "https://mareceh.com", "id", isNsfw = true, overrideVersionCode = 1),
```

Comment:
```suggestion
        SingleLang("Mareceh", "https://mareceh.com", "id", isNsfw = true, overrideVersionCode = 10),
```

Needs to be `10` because the old one was `23` (Madara's `baseVersionCode = 20` + Mangceh's `overrideVersionCode 
 = 3`), and since WpMangaStream's `baseVersionCode = 14`, `baseVersionCode + overrideVersionCode` should be `24` to it be an update and new version.